File: peek_plugin_generic_diagram_menu/_private/tuples/StringCapToggleActionTuple.py
from vortex.Tuple import addTupleType, TupleField
from vortex.TupleAction import TupleActionABC

from peek_plugin_generic_diagram_menu._private.PluginNames import genericDiagramMenuTuplePrefix


@addTupleType
class StringCapToggleActionTuple(TupleActionABC):
    __tupleType__ = genericDiagramMenuTuplePrefix + "StringCapToggleActionTuple"

    genericDiagramMenuId = TupleField()

File: peek_plugin_generic_diagram_menu/plugin-module/_private/tuples/StringCapToggleActionTuple.ts
                                       import {addTupleType, Tuple, TupleActionABC} from "@synerty/vortexjs";
import {genericDiagramMenuTuplePrefix} from "../PluginNames";

@addTupleType
export class StringCapToggleActionTuple extends TupleActionABC {
    static readonly tupleName = genericDiagramMenuTuplePrefix + "StringCapToggleActionTuple";

    genericDiagramMenuId: number;

    constructor() {
        super(StringCapToggleActionTuple.tupleName)
    }
}